go.mod

```diff
@@ -105,3 +105,5 @@ require (
 )
 
 replace mellium.im/sasl => github.com/mellium/sasl v0.2.1
+
+replace github.com/onflow/cadence => ./cadence
```

Comment:
we probably don't want to commit this, we need to change it to use cadence at specific tag